evm-adapters/src/sputnik/cheatcodes/mod.rs

```diff
@@ -61,6 +61,7 @@ ethers::contract::abigen!(
             record()
             accesses(address)(bytes32[],bytes32[])
             expectEmit(bool,bool,bool,bool)
+            mockCall(address,bytes,bytes)
```

Comment:
I was thinking we could also have `mockCall(address,bytes)` as an override, for checking functions w/o return data in a more user friendly way

Reply:
good points. how does abigen work with overrides?

Reply:
actually, i'm not sure i understand the first one, can you clarify? i don't see the use case. is it just to avoid a revert?

Reply:
> I was thinking we could also have mockCall(address,bytes) as an override, for checking functions w/o return data in a more user friendly way.

Kinda wish this was this was added. Would be useful for mocking functions that don't have return (external contract calls not in my implementation)